etcdserver/etcdserverpb/rpc.proto

```diff
@@ -394,6 +394,9 @@ message Member {
   // If the member is not started, client_URLs will be an zero length
   // string array.
   repeated string clientURLs = 5;
+  // server version of this member
+  // TODO: move this and IsLeader to "status" knob
+  string version = 6;
```

Comment:
i think we need to add 

```
// MemberStatus gets the status of the member with given ID
rpc MemberStatus(MemberStatusRequest) returns (MemberStatusResponse) {}
```
at line 76 inside cluster service.

For the MemberStatusRequest it should include the memberID. 
For the Response, it should include version for now. (it will include other detailed member status in the future)